src/content/docs/en/community-resources/content.mdx

```diff
@@ -0,0 +1,86 @@
+
+  ### Video Tutorials
+  - [Astro Crash Course in 20 minutes!](https://www.youtube.com/watch?v=zrPVTf761OI)
+  - [Astro Quick Start Course: Build an SSR Blog](https://www.youtube.com/watch?v=XoIHKO6AkoM)
+  - [How I created a movie application with Astro 3.x](https://www.youtube.com/watch?v=Gvr4WhgfP0w)
+  - [You may not actually understand Content Collections](https://www.youtube.com/watch?v=kwqlP6Pe0SI)
+  - [Everything You Need to Know about Astro](https://www.youtube.com/watch?v=rRxuVOutmFQ)
+  - [Build a custom blog platform with Astro and Apprwrite](https://www.youtube.com/watch?v=OERqwLy_reA)
+  - [Astro JS Portfolio Crash Course](https://www.youtube.com/watch?v=TwWvNK0yHjI)
+  - [Build a full stack blog with Astro](https://egghead.io/courses/build-a-full-stack-blog-with-astro-7ffcf9ec)
+
+
+## Recipes and Guides
+
+See guided examples of adding features to your Astro project, submitted by our community members!
+
+[Astro recipes](/en/recipes/) are typically short, focused how-to guides that walk a reader through completing a working example of a specific task. Recipes are a great way to add new features or behavior to your Astro project by following step-by-step instructions!
+
+Other guides might explain concepts related to an area of content, such as using images or working with MDX.
+
+:::tip[Add your own!]
+Have you written a recipe for Astro? [Edit this page](https://github.com/withastro/docs/edit/main/src/content/docs/en/community-resources/content.mdx) and add your link in the appropriate section!
+:::
+
+
+### CSS
+- [Use UnoCSS in Astro](https://www.elian.codes/blog/23-02-11-implementing-unocss-in-astro/)
+- [Add dark mode to Astro with Tailwind CSS](https://www.kevinzunigacuellar.com/blog/dark-mode-in-astro/)
+### Authentication
+- [Add Github OAuth with Lucia](https://lucia-auth.com/guidebook/github-oauth/astro)
+- [Add username and password authentication with Lucia](https://lucia-auth.com/guidebook/sign-in-with-username-and-password/astro)
+### RSS
+- [Use a dynamic filename when importing images](https://vaihe.com/blog/astro/astro-dynamic-image-prop/)
+- [Add an RSS feed to Astro using Storyblok's stories](https://straffesites.com/en/blog/rss-feed-astro-storyblok)
+- [How to Send Any Data to Your RSS Feed? A Guide with Astro + RSS](https://aritraroy.live/tutorial/blogs/2023/how-to-send-any-data-to-rss-feed/)
+### Images
+- [Add blog post images to your Astro RSS Feed](https://webreaper.dev/posts/astro-rss-feed-blog-post-images/)
+- [Generate Open Graph images for an Astro blog](https://techsquidtv.com/blog/generating-open-graph-images-for-astro/)
+### Editor Tooling
+- [Get VSCode, ESLint & Prettier working with Astro](https://patheticgeek.dev/blog/astro-prettier-eslint-vscode)
+- [Integrate Prettier with Astro and Tailwind CSS](https://straffesites.com/en/blog/integrate-prettier-astro-tailwindcss)
+### Animation
+- [Add animated page transitions with Swup](https://navillus.dev/blog/astro-plus-swup)
```

Comment:
Swup is something View Transitions provide with lesser setup, that being said this blogpost is outdated, I tried their instructions with Astro Starter template (could not make it work, even the repo they provide to `degit` seems unavailable😔)

Reply:
Ah, true this *is* a pretty old article. OK, if too old to be relevant, then we'll remove it. Thank you for checking!